addons/website_slides/wizard/slide_channel_invite.py

```diff
@@ -17,40 +17,40 @@ class SlideChannelInvite(models.TransientModel):
     _name = 'slide.channel.invite'
     _description = 'Channel Invitation Wizard'
 
-    @api.model
-    def _default_email_from(self):
-        if self.env.user.email:
-            return formataddr((self.env.user.name, self.env.user.email))
-        raise UserError(_("Unable to post message, please configure the sender's email address."))
-
-    @api.model
-    def _default_author_id(self):
-        return self.env.user.partner_id
-
     # composer content
-    subject = fields.Char('Subject')
-    body = fields.Html('Contents', default='', sanitize_style=True)
+    subject = fields.Char('Subject', compute='_compute_subject_and_body', readonly=False)
+    body = fields.Html('Contents', default='', sanitize_style=True, compute='_compute_subject_and_body', readonly=False)
```

Comment:
Let's go with `_compute_template_values` instead.
Also, don't you need to specify `store=True` as well?

Reply:
It acts just like a regular model except it's cleaned by a CRON every few minutes.
So I think you need the `store=True` indeed.